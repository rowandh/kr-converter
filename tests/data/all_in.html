
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-2-86303596</p></td>
                                <td height="26" ><p align="center">&nbsp;2024-09-09 오후 10:32:44</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>zxssdv (7 4 투 페어)</b></p></td>
<!--								<td ><p align="center">&nbsp;172354</p></td>   -->
								<td ><p align="center">&nbsp;172,354</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>3whj3jk21</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>J ♠6 [J 탑]  (다이) 
<br>* 시작 : [StageNo:86303596] [Credit:200,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:200]
<br>* NICKNAME:[3whj3jk21]
<br>* 앤티: -1,000원(199,000원)
<br>* 홀 카드딜: <font color='red'>◆</font>J(23) ♠6(5) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(<font color='red'>◆</font>J ♠6)]
<br>* 베팅: 다이 [0](199,000원) - 베팅순서: [0][6]
[4435ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>J♠6) C (♣4♠7♣J)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>J♠6) C (♣4♠7♣J) (<font color='red'>♥</font>6) (♠4)
<br>* 종료: WinMoney[0원] Credit[199,000원]
<br>* 결과: 패배 [족보:J 탑] [카드:<font color='red'>◆</font>J ♠6] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">199,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>bhrghthtr</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>9 <font color='red'>◆</font>3 ♣4 ♠7 ♣J [J 탑]  (다이) (Small Blind)
<br>* 시작 : [StageNo:86303596] [Credit:64,136원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[bhrghthtr]
<br>* 앤티: -1,000원(63,136원)
<br>* 홀 카드딜: <font color='red'>◆</font>9(21) <font color='red'>◆</font>3(15) [9 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:9 탑(<font color='red'>◆</font>9 <font color='red'>◆</font>3)]
<br>* 베팅: [블라인드:SMALL] [금액:1,000원] [Creadit:62,136원]
<br>* 베팅: 체크 -0원(62,136원) - 베팅순서: [0][8]
[273ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>9<font color='red'>◆</font>3) C (♣4♠7♣J)
<br>* 턴 시작: [플랍(1)] [족보:J 탑(♣J <font color='red'>◆</font>9 ♠7 ♣4 <font color='red'>◆</font>3)]
<br>* 베팅: 다이 [0](62,136원) - 베팅순서: [1][1]
[1503ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>9<font color='red'>◆</font>3) C (♣4♠7♣J) (<font color='red'>♥</font>6) (♠4)
<br>* 종료: WinMoney[0원] Credit[62,136원]
<br>* 결과: 패배 [족보:J 탑] [카드:♣J <font color='red'>◆</font>9 ♠7 ♣4 <font color='red'>◆</font>3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">62,136</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>zxssdv</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣10 ♣7 ♣4 ♠7 ♣J <font color='red'>♥</font>6 ♠4 [7 4 투 페어]  
<br>* 시작 : [StageNo:86303596] [Credit:78,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[zxssdv]
<br>* 앤티: -1,000원(77,000원)
<br>* 홀 카드딜: ♣10(48) ♣7(45) [10 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:10 탑(♣10 ♣7)]
<br>* 베팅: 콜 -1,000원(76,000원) - 베팅순서: [0][2]
[1042ms]
<br>* 커뮤니티 카드 딜: H(♣10♣7) C (♣4♠7♣J)
<br>* 턴 시작: [플랍(1)] [족보:7 원 페어(♠7 ♣7 ♣J ♣10 ♣4)]
<br>* 베팅: [하프] (37,500원) Credit(38,500원) - 베팅순서: [1][4]
[4672ms]
<br>* 베팅: [올인] (38,500원) Credit(0원) - 베팅순서: [1][9]
[1976ms]
<br>* 쇼다운 상태 진입: [턴:1]
<br>* 커뮤니티 카드 딜: H(♣10♣7) C (♣4♠7♣J) (<font color='red'>♥</font>6) (♠4)
<br>* 나가기예약:[1]
<br>* 종료: WinMoney[172,354원] Credit[172,354원]
<br>* 결과: 승리 [족보:7 4 투 페어] [카드:♠7 ♣7 ♣4 ♠4 ♣J]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>94,354</font></p></td>
											<td style="word-break:break-all"><p align="right">172,354</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>gtelu940hs</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠A ♠3 ♣4 ♠7 ♣J [A 탑]  (다이) 
<br>* 시작 : [StageNo:86303596] [Credit:300,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[bdfg34534]
<br>* 앤티: -1,000원(299,000원)
<br>* 홀 카드딜: ♠A(0) ♠3(2) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(♠A ♠3)]
<br>* 베팅: 콜 -1,000원(298,000원) - 베팅순서: [0][3]
[1054ms]
<br>* 커뮤니티 카드 딜: H(♠A♠3) C (♣4♠7♣J)
<br>* 턴 시작: [플랍(1)] [족보:A 탑(♠A ♣J ♠7 ♣4 ♠3)]
<br>* 베팅: 다이 [0](298,000원) - 베팅순서: [1][5]
[1094ms]
<br>* 커뮤니티 카드 딜: H(♠A♠3) C (♣4♠7♣J) (<font color='red'>♥</font>6) (♠4)
<br>* 종료: WinMoney[0원] Credit[298,000원]
<br>* 결과: 패배 [족보:A 탑] [카드:♠A ♣J ♠7 ♣4 ♠3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">298,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>Qqjwjqkqk</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠J ♣9 ♣4 ♠7 ♣J [J 원 페어]  (다이) 
<br>* 시작 : [StageNo:86303596] [Credit:97,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[qqjwjqkqk]
<br>* 앤티: -1,000원(96,000원)
<br>* 홀 카드딜: ♠J(10) ♣9(47) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(♠J ♣9)]
<br>* 베팅: 콜 -1,000원(95,000원) - 베팅순서: [0][1]
[4824ms]
<br>* 커뮤니티 카드 딜: H(♠J♣9) C (♣4♠7♣J)
<br>* 턴 시작: [플랍(1)] [족보:J 원 페어(♣J ♠J ♣9 ♠7 ♣4)]
<br>* 베팅: [풀] (15,000원) Credit(80,000원) - 베팅순서: [1][3]
[732ms]
<br>* 베팅: 다이 [0](80,000원) - 베팅순서: [1][8]
[4747ms]
<br>* 커뮤니티 카드 딜: H(♠J♣9) C (♣4♠7♣J) (<font color='red'>♥</font>6) (♠4)
<br>* 종료: WinMoney[0원] Credit[80,000원]
<br>* 결과: 패배 [족보:J 원 페어] [카드:♣J ♠J ♣9 ♠7 ♣4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-17,000</font></p></td>
											<td style="word-break:break-all"><p align="right">80,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>xbvnfgtrf</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>6 ♣3 [6 탑]  (다이) 
<br>* 시작 : [StageNo:86303596] [Credit:81,500원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[xbvnfgtrf]
<br>* 앤티: -1,000원(80,500원)
<br>* 홀 카드딜: <font color='red'>◆</font>6(18) ♣3(41) [6 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:6 탑(<font color='red'>◆</font>6 ♣3)]
<br>* 베팅: 다이 [0](80,500원) - 베팅순서: [0][5]
[222ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>6♣3) C (♣4♠7♣J)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>6♣3) C (♣4♠7♣J) (<font color='red'>♥</font>6) (♠4)
<br>* 종료: WinMoney[0원] Credit[80,500원]
<br>* 결과: 패배 [족보:6 탑] [카드:<font color='red'>◆</font>6 ♣3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">80,500</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>ckadmfheo</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠K <font color='red'>♥</font>5 [K 탑]  (다이) 
<br>* 시작 : [StageNo:86303596] [Credit:167,932원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[ckadmfheo]
<br>* 앤티: -1,000원(166,932원)
<br>* 홀 카드딜: ♠K(12) <font color='red'>♥</font>5(30) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(♠K <font color='red'>♥</font>5)]
<br>* 베팅: 다이 [0](166,932원) - 베팅순서: [0][4]
[2711ms]
<br>* 커뮤니티 카드 딜: H(♠K<font color='red'>♥</font>5) C (♣4♠7♣J)
<br>* 커뮤니티 카드 딜: H(♠K<font color='red'>♥</font>5) C (♣4♠7♣J) (<font color='red'>♥</font>6) (♠4)
<br>* 종료: WinMoney[0원] Credit[166,932원]
<br>* 결과: 패배 [족보:K 탑] [카드:♠K <font color='red'>♥</font>5] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">166,932</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>kvvsufhe</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣A ♣2 ♣4 ♠7 ♣J <font color='red'>♥</font>6 ♠4 [4 원 페어]  
<br>* 시작 : [StageNo:86303596] [Credit:152,615원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[kvvsufhe]
<br>* 앤티: -1,000원(151,615원)
<br>* 홀 카드딜: ♣A(39) ♣2(40) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(♣A ♣2)]
<br>* 베팅: 콜 -1,000원(150,615원) - 베팅순서: [0][7]
[831ms]
<br>* 커뮤니티 카드 딜: H(♣A♣2) C (♣4♠7♣J)
<br>* 턴 시작: [플랍(1)] [족보:A 탑(♣A ♣J ♠7 ♣4 ♣2)]
<br>* 베팅: [레이즈] (150,615원) Credit(0원) - 베팅순서: [1][6]
[3394ms]
<br>* 쇼다운 상태 진입: [턴:1]
<br>* 커뮤니티 카드 딜: H(♣A♣2) C (♣4♠7♣J) (<font color='red'>♥</font>6) (♠4)
# 공베팅 반환 [74,615원]<br>* 종료: WinMoney[0원] Credit[74,615원]
<br>* 결과: 패배 [족보:4 원 페어] [카드:♣4 ♠4 ♣A ♣J ♠7]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-78,000</font></p></td>
											<td style="word-break:break-all"><p align="right">74,615</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>glkfjhl12</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣K <font color='red'>◆</font>7 ♣4 ♠7 ♣J [7 원 페어]  (다이) (Big Blind)
<br>* 시작 : [StageNo:86303596] [Credit:415,051원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[glkfjhl12]
<br>* 앤티: -1,000원(414,051원)
<br>* 홀 카드딜: ♣K(51) <font color='red'>◆</font>7(19) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(♣K <font color='red'>◆</font>7)]
<br>* 베팅: [블라인드:BIG] [금액:1,000원] [Creadit:413,051원]
<br>* 베팅: 체크 -0원(413,051원) - 베팅순서: [0][9]
[1242ms]
<br>* 커뮤니티 카드 딜: H(♣K<font color='red'>◆</font>7) C (♣4♠7♣J)
<br>* 턴 시작: [플랍(1)] [족보:7 원 페어(♠7 <font color='red'>◆</font>7 ♣K ♣J ♣4)]
<br>* 베팅: 체크 -0원(413,051원) - 베팅순서: [1][2]
[1703ms]
<br>* 베팅: 다이 [0](413,051원) - 베팅순서: [1][7]
[1726ms]
<br>* 커뮤니티 카드 딜: H(♣K<font color='red'>◆</font>7) C (♣4♠7♣J) (<font color='red'>♥</font>6) (♠4)
<br>* 종료: WinMoney[0원] Credit[413,051원]
<br>* 결과: 패배 [족보:7 원 페어] [카드:♠7 <font color='red'>◆</font>7 ♣K ♣J ♣4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">413,051</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-2-86303596">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


