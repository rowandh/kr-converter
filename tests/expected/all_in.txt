PokerStars Hand #15286303596:  Hold'em No Limit ($1000/$1000) - 2024/09/09 22:32:44 KST
Table 'Table 1' 9-max Seat #7 is the button
Seat 1: Qqjwjqkqk ($97000 in chips)
Seat 2: zxssdv ($78000 in chips)
Seat 3: gtelu940hs ($300000 in chips)
Seat 4: ckadmfheo ($167932 in chips)
Seat 5: xbvnfgtrf ($81500 in chips)
Seat 6: 3whj3jk21 ($200000 in chips)
Seat 7: kvvsufhe ($152615 in chips)
Seat 8: bhrghthtr ($64136 in chips)
Seat 9: glkfjhl12 ($415051 in chips)
Qqjwjqkqk: posts the ante $1000
zxssdv: posts the ante $1000
gtelu940hs: posts the ante $1000
ckadmfheo: posts the ante $1000
xbvnfgtrf: posts the ante $1000
3whj3jk21: posts the ante $1000
kvvsufhe: posts the ante $1000
bhrghthtr: posts the ante $1000
glkfjhl12: posts the ante $1000
bhrghthtr: posts small blind $1000
glkfjhl12: posts big blind $1000
*** HOLE CARDS ***
Qqjwjqkqk: calls $1000
zxssdv: calls $1000
gtelu940hs: calls $1000
ckadmfheo: folds
xbvnfgtrf: folds
3whj3jk21: folds
kvvsufhe: calls $1000
bhrghthtr: checks
glkfjhl12: checks
*** FLOP *** [4c 7s Jc]
bhrghthtr: folds
glkfjhl12: checks
Qqjwjqkqk: bets $15000
zxssdv: raises $22500 to $37500
gtelu940hs: folds
kvvsufhe: raises $113115 to $150615 and is all-in
glkfjhl12: folds
Qqjwjqkqk: folds
zxssdv: calls $38500 and is all-in
Uncalled bet ($74615) returned to kvvsufhe
*** TURN *** [4c 7s Jc] [6h]
*** RIVER *** [4c 7s Jc] [6h] [4s]
zxssdv collected $172354 from pot
*** SUMMARY ***
Total pot $182000 | Rake $9646
Board [4c 7s Jc 6h 4s]
Seat 1: Qqjwjqkqk mucked [Js 9c]
Seat 2: zxssdv showed [Tc 7c] and won ($172354)
Seat 3: gtelu940hs mucked [As 3s]
Seat 4: ckadmfheo mucked [Ks 5h]
Seat 5: xbvnfgtrf mucked [6d 3c]
Seat 6: 3whj3jk21 mucked [Jd 6s]
Seat 7: kvvsufhe showed [Ac 2c] and lost
Seat 8: bhrghthtr mucked [9d 3d]
Seat 9: glkfjhl12 mucked [Kc 7d]